src/components/Form/formStyling.tsx

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import styled from "styled-components";
+
+export const CenterComponent = styled.div`
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    margin: 2em;
+`;
+
+export const StyledForm = styled.form`
+    width: 90%;
+    display: flex;
+    justify-content: center;
+    align-content: center;
+    flex-direction: column;
+`;
+
+export const StyledCard = styled.div`
+    display: flex;
+    flex-direction: column;
+    padding: 2em;
+    margin: 2em;
+    width: 100%;
+    height: 80%;
+    border-radius: 10px;
+    background-color: ${(props) => props.theme.surfaceBackgroundColor};
+    color: ${(props) => props.theme.surfaceForegroundColor};
+
+    div {
+        width: 100%;
+        margin: 0.15em 0;
+    }
+`;
+
+export const Text = styled.p`
+    padding-bottom: 1em;
+`;
+
+export const Subheading = styled.h2`
+    padding-bottom: 1em;
+`;
+
+export const Heading = styled.h1`
+    padding-bottom: 1em;
+`;
+
+const RequiredAsterisk = styled.span`
+    color: ${(props) => props.theme.errorColor};
+
+    &:before {
+        content: "*";
+    }
+`;
+
+export const ErrorText = styled(Text)`
+    color: ${(props) => props.theme.errorColor};
+    margin-bottom: 3em;
+    text-align: center;
+`;
+
+export const StyledButton = styled.button`
```

Comment:
This is quite a generic name if it's being exported, for example I know I've used similar names throughout stuff I've coded.